cli/src/main/java/org/infinispan/cli/commands/SecurityActions.java

```diff
@@ -1,34 +1,21 @@
 package org.infinispan.cli.commands;
 
-import java.security.AccessController;
-import java.security.PrivilegedAction;
 import java.security.Provider;
 
 /**
  * SecurityActions for the org.infinispan.cli.commands package.
  * <p>
- * Do not move. Do not change class and method visibility to avoid being called from other {@link
- * java.security.CodeSource}s, thus granting privilege escalation to external code.
+ * Do not move. Do not change class and method visibility to avoid being called from other
+ * {@link java.security.CodeSource}s, thus granting privilege escalation to external code.
  *
  * @author Tristan Tarrant <tristan@infinispan.org>
  * @since 12.0
  */
 final class SecurityActions {
```

Comment:
is it worth keeping these classes (`SecurityActions`)?

Reply:
I'm removing them all